README.md

```diff
@@ -1,191 +1,154 @@
+- Here  is the link to [kanban board initial state](https://user-images.githubusercontent.com/108806646/223073503-281902af-2be4-44a1-840c-394ca01a7447.png)
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+<!-- React Frontend -->
+### <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/539px-React-icon.svg.png" width="20" height="20"/>  <a href="https://github.com/DaveZag/meal-master-frontend">Meal-Master-Frontend</a>
 
-## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+## 🚀 **Live demo** <a id="live-demo"></a>
 
-- [Live Demo Link](https://google.com)
+- [Live Demo](https://master-meal.netlify.app/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
 
-## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
+## 💻 **Getting started** <a id="getting-started"></a>
+
+> This project can be used by anyone for any good purpose.
 
 To get a local copy up and running, follow these steps.
 
-### Prerequisites
+### **Pre requisites**
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+> - Ruby and Rails on you operating system eg. for  [Windows](https://gorails.com/setup/windows/10) or [Linux](https://gorails.com/setup/ubuntu/20.04)
+> - React.js
+> - [Install nodejs](https://nodejs.org/ca/blog/release/v16.19.0/)
+> - [PostgreSQL](https://www.postgresql.org/) database installed
+> - [Install Git](https://git-scm.com/)
+> - Install a code editor of your choice
+> - Knwoledge of javascript and react
+> - Github account is required
 
-### Setup
+### **Setup**
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/DaveZag/meal-master-backend.git
 ```
---->
 
-### Install
+### **Install**
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  cd meal-master-backend # Change directory into project folder
+  bundle install # Installs project dependencies
 ```
---->
+### Important steps after installing project to local machine
+1) Please run rails db:create and rails db:migrate. 
+2) Check Ruby version and database.yml file. ( Insert your password if there is required)
 
-### Usage
+### **Usage**
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  rails s # for running the project in development mode
 ```
---->
-
-### Run tests
+### **Run tests**
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
 ```sh
-  bin/rails test test/models/article_test.rb
+  rubocop -A # for checking and correcting linter errors
+  bin/rails rspec spec
```

Comment:
- [ ] This command `bin/rails rspec spec` does not work for me:

![image](https://user-images.githubusercontent.com/57421823/225816384-41a24772-32d8-464a-938c-3a39365c7280.png)

Also if I just write `rspec ./spec`:

![image](https://user-images.githubusercontent.com/57421823/225816480-2ceeac80-4496-4212-96d4-3f1e81635ff7.png)

This is because your tests are swagger tests.

Either add some unit tests, or change your command to `rake rswag`:

![image](https://user-images.githubusercontent.com/57421823/225816637-b99c0ea0-6683-4f33-88d1-8480400ded96.png)

That way, your users will never get a failed test. 💪